packages/tracerbench/src/benchmarks/initial-render-metric.ts

```diff
@@ -192,6 +194,7 @@ export default class InitialRenderMetric {
     const markers = this.markers;
     const phaseEvents: ITraceEvent[] = [];
     let eventIdx = 0;
+    fs.writeFileSync('events_output.json', JSON.stringify(events));
```

Comment:
it isn't clear how this file is consumed and if this a necessary output. Will you confirm?

Reply:
oh interesting i don't remember making this change. Will revert this one